Makefile

```diff
@@ -326,10 +326,10 @@ define deploy-local-cluster
 		--set k8gb.log.level=$(LOG_LEVEL)
 
 	@echo "\n$(YELLOW)Deploy Ingress $(NC)"
-	helm repo add --force-update stable https://charts.helm.sh/stable
+	helm repo add nginx-stable https://kubernetes.github.io/ingress-nginx
```

Comment:
`--force-update` was added to not fail locally when the repo is already present. Why was it removed?

Reply:
```shell
 > helm repo add nginx-stable https://kubernetes.github.io/ingress-nginx
"nginx-stable" already exists with the same configuration, skipping```

Reply:
Option is deprecated and no longer needed

Reply:
@k0da since which helm version?

Reply:
```
$ helm repo add nginx-stable https://kubernetes.github.io/ingress-nginx
Error: repository name (nginx-stable) already exists, please specify a different name
$ echo $?
1
```

Reply:
@ytsarev I just reverted it, since option is still there.